lib/structures/User.js

```diff
@@ -50,7 +52,18 @@ class User extends Base {
     }
 
     get avatarURL() {
-        return this.avatar ? `${CDN_URL}/avatars/${this.id}/${this.avatar}.${this.avatar.startsWith("a_") ? "gif" : "jpg"}` : this.defaultAvatarURL;
+        return this.avatar ? `${CDN_URL}/avatars/${this.id}/${this.avatar}.${this.avatar.startsWith("a_") ? "gif" : this._client.options.defaultImageFormat}${this._client.options.defaultImageSize == 128 ? "" : `?size=${this._client.options.defaultImageSize}`}` : this.defaultAvatarURL;
+    }
+
+    /**
+    * Get the user's avatar with the given format and size
+    * @arg {String} [format=default] The filetype of the avatar (jpg, png, gif, or webp)
+    * @arg {Number} [size=default] The size of the avatar (128, 256, 512, 1024, 2048)
+    */
+    dynamicAvatarURL(format, size) {
+        if (format === undefined || !ImageFormats.includes(format.toLowerCase())) format = this._client.options.defaultImageFormat;
+        if (size === undefined || !ImageSizes.includes(size)) size = this._client.options.defaultImageSize;
```

Comment:
Use `if(!~array.indexOf(item))` instead of `if(!array.includes(item))`

If statements should be formatted like the rest of the lib code
```
if(bool) {
    // thing
}
```